### ISSUE TITLE
File I/O

### ISSUE THREAD

**Issue by rzuckerm** [never-lang/never#57]
Does Never support reading and write files? I can't seem to find anything in the documentation about this?

**Comment by smaludzi** [comment]
No. There is such module right now. I think you could try to use foreign function interface or add builtin functions.
